roofit/roofitcore/test/testRooFuncWrapper.cxx

```diff
@@ -327,3 +327,39 @@ TEST(RooFuncWrapper, GaussianNormalized)
       EXPECT_NEAR(getNumDerivative(*pdf, static_cast<RooRealVar &>(*paramsGauss[i]), normSet), dMyGauss[i], 1e-8);
    }
 }
+
+TEST(RooFuncWrapper, GaussianNormalizedAutoCodeSquash)
```

Comment:
```suggestion
TEST(RooFuncWrapper, GaussianNormalized)
```
You can rename this to the name that the old test with the mock code had and delete the old test, plus all the code starting on line 180 or so that I wrote to demonstrate how the integral handling should work. It's not needed anymore now that you have implemented the code squashing properly.